src/shared/lib/oql/oql_basic.pegjs

```diff
@@ -1,23 +1,25 @@
 start
-	= Query
+	= SuperQuery
 	/ sp { return false; }
 
-br	= [\n] b:br
-        / ";" b:br
-        / [\n]
-	/ ";"
-
 NaturalNumber = number:[0-9]+ { return number.join("");}
 Number = "-" number: Number { return "-"+number;}
         / whole_part:NaturalNumber "." decimal_part:NaturalNumber { return whole_part + "." + decimal_part;}
         / "." decimal_part:NaturalNumber { return "."+decimal_part;}
         / whole_part:NaturalNumber {return whole_part;}
 String = word:[-_.@/a-zA-Z0-9*]+ { return word.join("") }
 AminoAcid = letter:[GPAVLIMCFYWHKRQNEDST] { return letter; }
+AnyString = anyString:[^"]+ { return anyString.join("") }
 
 sp = space:[ \t\r]+
 msp = space:[ \t\r]*
 
+zmbs = zero_or_more_breaks_and_spaces:[; \t\r\n]*
+ombs = one_or_more_breaks_and_spaces:[; \t\r\n]+
```

Comment:
since here any whitespace is okay, how about `[;\s]` instead?

Reply:
I'm not the author of this part, but I believe that it's not actually a regular expression, but rather a [special, less advanced pegjs syntax](https://pegjs.org/documentation#grammar-syntax-and-semantics-parsing-expression-types)—
> Match one character from a set and return it as a string. The characters in the list can be escaped in exactly the same way as in JavaScript string.

That suggests that `\s` may not be supported.

Reply:
oh I see